src/pages/Adventure/index.tsx

```diff
@@ -99,10 +102,15 @@ export default function Adventure(): JSX.Element | null {
 
                 {summoners ? (
                     summoners.length > 0 ? (
-                        <div className="grid grid-cols-1 lg:grid-cols-2 xl:grid-cols-3 w-10/12 mx-auto mt-10 gap-4">
-                            {summoners.map((summoner) => {
-                                return <SummonerAdventureCard key={summoner.id} summoner={summoner} />
-                            })}
+                        <div className="flex flex-col items-center mt-10">
+                            <p className="text-lg text-white text-left container ml-20 lg:-ml-16 mb-5">
+                                Current Time: {current_time.toUTCString()}
+                            </p>
+                            <div className="grid grid-cols-1 lg:grid-cols-2 xl:grid-cols-3 w-10/12 mx-auto gap-4">
+                                {summoners.map((summoner) => {
+                                    return <SummonerAdventureCard key={summoner.id} summoner={summoner} />
+                                })}
+                            </div>
```

Comment:
Time is not properly centered, and I think it is not required

Reply:
The reason why I've added this is because I noticed that your "next adventure in" timer is not refreshing in real time so I added an interval to display the current time and also to ensure the "next adventure in" timer is refreshed too. I can remove it if you want.

Reply:
Ok, I see. 
I actually don't see the point of having the current time, but the auto-update time in "next adventure in" is really cool!

Reply:
I understand. I'll remove the current time in adventure page now